src/app/add-item/add-item.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="container dialog-modal">
+    <h2 class="dialogTitle" mat-dialog-title>{{ title }}</h2>
+    <mat-dialog-content [formGroup]="dialogForm">
+        <mat-form-field>
+            <input matInput class="typedValueColor name-input" placeholder="Action Item Name" formControlName="name" />
+            <mat-error *ngIf="dialogForm.get('name').hasError('required')" class="invalid-feedback">
+                Name is required.
+            </mat-error>
+        </mat-form-field>
+        <mat-form-field>
+            <mat-select class="project-input" formControlName="project" placeholder="Project">
+                <mat-option *ngFor="let project of projects" [value]="project">
+                    {{ project.name }}
+                </mat-option>
+            </mat-select>
+            <mat-error *ngIf="dialogForm.get('project').hasError('required')" class="invalid-feedback" class="invalid-feedback">
```

Comment:
Same as above - https://material.angular.io/components/form-field/overview#error-messages

Reply:
You don't need to apply `invalid-feedback` class twice.